django/core/settings.py

```diff
@@ -9,6 +9,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'django_light',  # 3rd party, but must be before django.contrib.admin
```

Comment:
Is this an app that I should be looking into (I've not come across it before)?

Reply:
It's just to make the admin dashboard always appear in light mode. The dashboard chooses a light/dark theme based on the OS's settings. I have my OS as dark but most have it as light, so I use this to keep it consistent so when sending screenshots, giving demos, etc. it looks the same on my screen as the researchers' so it's easier for them to learn how to use it. Not a compulsory thing, just my preference.